README.md

```diff
@@ -0,0 +1,199 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b> 💰 Appointment App Back-end 💸 </b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [💸 Rails APi App ](#-appointment-app-)
+- [🛠 Built With ](#-built-with-)
+  - [📌 Tech Stack ](#-tech-stack-)
+- [🎲 Key Features ](#-key-features-)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 💸 Car Api <a name="about-project"></a>
```

Comment:
You should eventually put a live demo link. Also maybe mention the A?P?I documentation somehwere in your README

Reply:
BTW, this is a GREAT project, really good job, just really want you to add something about the API documentation in your README, otherwise you'd have gotten the 👍

Reply:
Thanks for the feedback @mcrd25., I have made the change requested